app/models/jobsModel.js

```diff
@@ -148,6 +148,23 @@ async function getByMinutesInRange(minMinutes, maxMinutes){
         console.error("Error: failed to dequeue jobs", e);
     }
 }
+/**
+ * Deletes a certain job given its ID
+ * 
+ * @param job_id 
+ */
+async function deleteJob(job_id){
```

Comment:
Use the updateById function instead which is already made.

Reply:
Addressed, thanks!